Docker run fails with weird error...

The Dockerfile is super simple, file `/app/main` is here, exec flag is set for everyone... Don't know what's going on.

```
docker run --name iam --rm mozilla/iam
panic: standard_init_linux.go:178: exec user process caused "no such file or directory" [recovered]
	panic: standard_init_linux.go:178: exec user process caused "no such file or directory"

goroutine 1 [running, locked to thread]:
panic(0x564503e34300, 0xc42013c770)
	/usr/lib/go-1.7/src/runtime/panic.go:500 +0x1a1
github.com/opencontainers/runc/vendor/github.com/urfave/cli.HandleAction.func1(0xc420093748)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/vendor/github.com/urfave/cli/app.go:478 +0x247
panic(0x564503e34300, 0xc42013c770)
	/usr/lib/go-1.7/src/runtime/panic.go:458 +0x243
github.com/opencontainers/runc/libcontainer.(*LinuxFactory).StartInitialization.func1(0xc420093198, 0xc420022080, 0xc420093238)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/libcontainer/factory_linux.go:259 +0x18f
github.com/opencontainers/runc/libcontainer.(*LinuxFactory).StartInitialization(0xc4200526e0, 0x5645041f3520, 0xc42013c770)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/libcontainer/factory_linux.go:277 +0x353
main.glob..func8(0xc420096780, 0x0, 0x0)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/main_unix.go:26 +0x66
reflect.Value.call(0x564503e1f080, 0x564503eaaa70, 0x13, 0x564503e7d57b, 0x4, 0xc420093708, 0x1, 0x1, 0x564503c15bb8, 0x564503e733e0, ...)
	/usr/lib/go-1.7/src/reflect/value.go:434 +0x5c8
reflect.Value.Call(0x564503e1f080, 0x564503eaaa70, 0x13, 0xc420093708, 0x1, 0x1, 0x564504209460, 0xc4200936e8, 0x564503c1eb96)
	/usr/lib/go-1.7/src/reflect/value.go:302 +0xa4
github.com/opencontainers/runc/vendor/github.com/urfave/cli.HandleAction(0x564503e1f080, 0x564503eaaa70, 0xc420096780, 0x0, 0x0)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/vendor/github.com/urfave/cli/app.go:487 +0x1e0
github.com/opencontainers/runc/vendor/github.com/urfave/cli.Command.Run(0x564503e7d747, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x564503e8eb6b, 0x51, 0x0, ...)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/vendor/github.com/urfave/cli/command.go:191 +0xc3b
github.com/opencontainers/runc/vendor/github.com/urfave/cli.(*App).Run(0xc4200dc000, 0xc42000c120, 0x2, 0x2, 0x0, 0x0)
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/vendor/github.com/urfave/cli/app.go:240 +0x611
main.main()
	/build/runc-HXIwmv/runc-1.0.0~rc2+docker1.12.6/_build/src/github.com/opencontainers/runc/main.go:137 +0xbd6
Makefile:11: recipe for target 'docker-run' failed
make: *** [docker-run] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@ GO_BINDATA := $(GOPATH)/bin/go-bindata
 DATA_FILES := ./utilities/openapi.yaml ./utilities/contribute.yaml
 
 main: utilities/bindata.go *.go utilities/*.go warden/*.go
-	go build -o main *.go
+	CGO_ENABLED=0 go build -o main *.go
 
 $(GO_BINDATA):
 	go get github.com/jteeuwen/go-bindata/...

